src/corelibs/U2Algorithm/src/temperature/methods/RoughTempCalcCmdFactory.cpp

```diff
@@ -24,10 +24,10 @@
 
 namespace U2 {
 
-const QString RoughTempCalcCmdFactory::ID = "Rough";
+const char* RoughTempCalcCmdFactory::ID = QT_TR_NOOP("Rough");
```

Comment:
Do not mix IDs and Display names.

Reply:
Now this is the same and it won't be easy to change that behavior. The only thing I can is to rename "ID" with "NAME"

Reply:
This is not a name but ID: you can store it into settings (as a last selected Tm algorithm id), use it in serialized WD schemas.

If support of the pair: visual name + persistent ID is too complex you can drop this part (translation of ID) from the patch and I will approve. Next I will help to make it the right way: with a separate ID & display text

Reply:
It would be difficult to fix now

Reply:
I see no technical difficulty about this fix. 
You can drop it translation of ID from the patch and once merged I will add a patch on top of it.

Reply:
OK